### PR TITLE
chore: add office hour file for 290923

### PR DESCRIPTION
**Pull request by almadigabor**
fixes https://github.com/eclipse-tractusx/sig-infra/issues/289

### CODE REVIEW

**Review by FaGru3n — APPROVED** [review]
LGTM

File: blog/2023-09-29-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+### Security
+
+- A Tractus-X release guide will be created. This will contain information about secrets, Trivy container, KICS infrastructure scanning. Security configuration will also be included for tools like Kubernetes and Postgresql.
+- SSI security topics are becoming more important as EDC is already having an SSI module.
+
+### FOSS
+
+- Tomasz Barwicki got elected as committer. The participation rate on the voting was really high and the committers are reminded that it is their duty to vote and be active.
```
Comment (FaGru3n) [inline blog/2023-09-29-OfficeHours.md:21]: you could add the GitHub profile reference. https://github.com/tomaszbarwicki

Comment (almadigabor) [reply blog/2023-09-29-OfficeHours.md]: done

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**